nautobot/ui/public/index.html

```diff
@@ -29,14 +32,13 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>{{ settings.BRANDING_TITLE }}</title>
+    <script defer="defer" src="{% static 'js/main.js' %}"></script>
+    <link href="{% static 'css/main.css' %}" rel="stylesheet">
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
     <div id="legacy-content" style="display:none">
-      {% load static %}
-      {% load helpers %}
-      {% load plugins %}
       {% block content %}{% endblock content %}
     </div>
     <!--
```

Comment:
Can/should the logo files below be symlinked to their equivalents in `nautobot/project-static`? Not blocking for this PR.

Reply:
Yeah kind of. We have a bigger fish to to fry when it comes to having Django serve the static files which includes images, using the manifest static files storage. So we'll cover that then.